content/.clang-format

```
@@ -0,0 +1,6 @@
+BasedOnStyle:    WebKit
+TabWidth:        8
+IndentWidth:     8
+UseTab:          Always
+ColumnLimit:     0
+PointerAlignment: Right
```

```suggestion
PointerAlignment: Right
IncludeCategories:
  - Regex:		'^<.*>'
    Priority:		0
  - Regex:              '^".*"'
    Priority:           1
```
This moves `<>` headers before `""`. I think it's more sensible like this.

---

I was on your boat, but seems other people thing it should be otherwise:

* https://stackoverflow.com/a/2762596/4804196
* https://softwareengineering.stackexchange.com/a/325565/389332
* https://blog.knatten.org/2010/07/01/the-order-of-include-directives-matter/

---

> other people

The worst this world has to offer. Ok, then leave it as it is.